### ISSUE TITLE
Casting  EnumDeclaration to ClassorInterfaceDeclaration crashes specimin

### ISSUE THREAD

**Issue by tahiat** [kelloggm/specimin#76]
command: 
`collect/Maps.java" --targetMethod "com.google.common.collect.Maps#difference(Map<? extends K, ? extends V>, Map<? extends K, ? extends V>, Equivalence<? super V>)"'`

Error: 

```
Exception in thread "main" java.lang.ClassCastException: class com.github.javaparser.ast.body.EnumConstantDeclaration cannot be cast to class com.github.javaparser.ast.body.ClassOrInterfaceDeclaration (com.github.javaparser.ast.body.EnumConstantDeclaration and com.github.javaparser.ast.body.ClassOrInterfaceDeclaration are in unnamed module of loader 'app')
        at org.checkerframework.specimin.UnsolvedSymbolVisitor.visit(UnsolvedSymbolVisitor.java:591)
        at org.checkerframework.specimin.UnsolvedSymbolVisitor.visit(UnsolvedSymbolVisitor.java:74)
        at com.github.javaparser.ast.body.MethodDeclaration.accept(MethodDeclaration.java:98)
        at com.github.javaparser.ast.visitor.ModifierVisitor.visit(ModifierVisitor.java:1129)
        at com.github.javaparser.ast.visitor.ModifierVisitor.visit(ModifierVisitor.java:51)
        at com.github.javaparser.ast.NodeList.accept(NodeList.java:277)
        at com.github.javaparser.ast.visitor.ModifierVisitor.modifyList(ModifierVisitor.java:1168)
        at com.github.javaparser.ast.visitor.ModifierVisitor.visit(ModifierVisitor.java:428)
        at com.github.javaparser.ast.visitor.ModifierVisitor.visit(ModifierVisitor.java:51)
        at com.github.javaparser.ast.body.EnumConstantDeclaration.accept(EnumConstantDeclaration.java:87)
        at com.github.javaparser.ast.visitor.ModifierVisitor.visit(ModifierVisitor.java:1129)
        at com.github.javaparser.ast.visitor.ModifierVisitor.visit(ModifierVisitor.java:51)
        at com.github.javaparser.ast.NodeList.accept(NodeList.java:277)
        at com.github.javaparser.ast.visitor.ModifierVisitor.modifyList(ModifierVisitor.java:1168)
        at com.github.javaparser.ast.visitor.ModifierVisitor.visit(ModifierVisitor.java:445)
        at com.github.javaparser.ast.visitor.ModifierVisitor.visit(ModifierVisitor.java:51)
        at com.github.javaparser.ast.body.EnumDeclaration.accept(EnumDeclaration.java:80)
        at com.github.javaparser.ast.visitor.ModifierVisitor.visit(ModifierVisitor.java:1129)
        at com.github.javaparser.ast.visitor.ModifierVisitor.visit(ModifierVisitor.java:51)
        at com.github.javaparser.ast.NodeList.accept(NodeList.java:277)
        at com.github.javaparser.ast.visitor.ModifierVisitor.modifyList(ModifierVisitor.java:1168)
        at com.github.javaparser.ast.visitor.ModifierVisitor.visit(ModifierVisitor.java:243)
        at org.checkerframework.specimin.UnsolvedSymbolVisitor.visit(UnsolvedSymbolVisitor.java:325)
        at org.checkerframework.specimin.UnsolvedSymbolVisitor.visit(UnsolvedSymbolVisitor.java:74)
        at com.github.javaparser.ast.body.ClassOrInterfaceDeclaration.accept(ClassOrInterfaceDeclaration.java:92)
        at com.github.javaparser.ast.visitor.ModifierVisitor.visit(ModifierVisitor.java:1129)
        at com.github.javaparser.ast.visitor.ModifierVisitor.visit(ModifierVisitor.java:51)
        at com.github.javaparser.ast.NodeList.accept(NodeList.java:277)
        at com.github.javaparser.ast.visitor.ModifierVisitor.modifyList(ModifierVisitor.java:1168)
        at com.github.javaparser.ast.visitor.ModifierVisitor.visit(ModifierVisitor.java:306)
        at com.github.javaparser.ast.visitor.ModifierVisitor.visit(ModifierVisitor.java:51)
        at com.github.javaparser.ast.CompilationUnit.accept(CompilationUnit.java:127)
        at org.checkerframework.specimin.SpeciminRunner.performMinimization(SpeciminRunner.java:131)
        at org.checkerframework.specimin.SpeciminRunner.main(SpeciminRunner.java:64)

> Task :run FAILED

```

Project [here](https://drive.google.com/file/d/1fYi9o6XrBjVLXrW5VT6giS5GPMj1DX0b/view?usp=sharing)

**Comment by LoiNguyenCS** [comment]
I've created PR #81 to handle this issue.

**Comment by kelloggm** [comment]
Fixed by #81.
